saleor/order/utils.py

```diff
@@ -13,12 +13,23 @@
 from ..extensions.manager import get_extensions_manager
 from ..order import OrderStatus
 from ..order.models import Order, OrderLine
-from ..product.utils import allocate_stock, deallocate_stock, increase_stock
 from ..product.utils.digital_products import get_default_digital_content_settings
 from ..shipping.models import ShippingMethod
+from ..stock.availability import check_stock_quantity
+from ..stock.management import allocate_stock, deallocate_stock, increase_stock
 from . import events
 
 
+def get_order_country(order: Order) -> str:
+    """Return country to which order will be shipped."""
+    address = order.billing_address
+    if order.is_shipping_required():
+        address = order.shipping_address
+    if address is None:
```

Comment:
IMO we should try to fetch `order.billing_address` first, then we can try with `DEFAULT_COUNTRY`

Reply:
I'm not sure I understand that - we are checking `billing address` at the beginning.